conf/application.conf

```diff
@@ -166,7 +166,8 @@ limits {
   maxIdentificationAuthorisations = 9
   maxIncidents = 9
   maxSeals = 99
-  maxTransportEquipments = 9999
+  maxNumberOfItems = 10
+  maxTransportEquipments = 10
```

Comment:
9999 for transport equipments

Reply:
Goods items is also 9999 by the looks of it

Reply:
Yeah I did this for testing purposes since it takes so long to generate all 9999 test cases